packages/sanity/src/structure/panes/documentList/sheetList/i18n/resources.ts

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Curious about this file colocation, added a question internally, I think it should be part of `packages/sanity/src/structure/i18n/resources.ts` but let's confirm before doing any change